edx/ux-pattern-library
# PL Doc: Set current nav item to is-current on page load.

Hi @talbs and @AlasdairSwan,

Another super small enhancement. Feel free to use or not. 

**Problem**
When loading a page from permalink, the current nav item is not set.

**Solution**
Compare the window.location.pathname to each nav item in succession, until we reach the one which matches. Then return false to stop the loop.

**How to test**
Check out this branch, navigate to a permalink, and you will note that the current nav item is highlighted.

Thanks!

Jamie
